src/ISDataMappings.cpp

```diff
@@ -1080,9 +1080,9 @@ static void PopulateISEventMappings(map_name_to_info_t mappings[DID_COUNT])
     ADD_MAP(m, totalSize, "Senders serial number", senderSN, 0, DataTypeUInt32, uint32_t, 0);
     ADD_MAP(m, totalSize, "Sender hardware type", senderHdwType, 0, DataTypeUInt16, uint16_t, 0);
 
-    ADD_MAP(m, totalSize, "protocol", protocol, 0, DataTypeUInt16, uint16_t, 0);
+    ADD_MAP(m, totalSize, "Protocol", protocol, 0, DataTypeUInt16, uint16_t, 0);
     ADD_MAP(m, totalSize, "Priority", priority, 0, DataTypeUInt8, uint8_t, 0);
-    ADD_MAP(m, totalSize, "length", length, 0, DataTypeUInt16, uint16_t, 0);
+    ADD_MAP(m, totalSize, "Length", length, 0, DataTypeUInt16, uint16_t, 0);
     ADD_MAP(m, totalSize, "data", data, 0, DataTypeString, uint8_t[MEMBERSIZE(MAP_TYPE, data)], 0);
```

Comment:
@tmcd249 I think it is better (more helpful) to use the exact variable names here.  (My opinion but not essential)